### ISSUE TITLE
Vampire Ancient ceremonial armor no sprite

### ISSUE THREAD

**Issue by AtomicGalaxy** [Blackstone-SS13/BLACKSTONE#379]
What it says on the tin, Ancient ceremonial armor that vampyre lord spawns with has no sprite, Apparently the sprite DOES exist in the code it just needs to have _F or _M on the end of it.

**Comment by Osmos123** [comment]
Yep, I had to do many renaming and additions of sleeves for the chest armor. It should be good now and appear in game.
